### PR TITLE
Create cmip6-tasmax-monthly-annual-max (no E&A capability)

### PR DESCRIPTION
**Pull request by freitagb**
## Why are you creating this Pull Request?

- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

### CODE REVIEW

File: datasets/cmip6-tasmax-monthly-annual-max.data.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    ## Data Summary  
+    * **Temporal Extent:** 1950 - 2100
+    * **Temporal Resolution:** Annual
+    * **Spatial Extent:** Global 
+    * **Spatial Resolution:** 0.25 degrees x 0.25 degrees
+    * **Data Units:** Degrees Farenheit
```
Comment (j08lue) [inline datasets/cmip6-tasmax-monthly-annual-max.data.mdx:86]:
```suggestion
    * **Data Units:** Degrees Fahrenheit
```

File: datasets/cmip6-tasmax-monthly-annual-max.data.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    * **Data Type:** Research
+
+    This dataset provides the Annual Maximum Near-Surface Air Temperature from 1950 - 2100. The projections are provided for the four different SSP standard scenarios ([SSP126, SSP245, SSP370, SSP585](https://www.dkrz.de/en/communication/climate-simulations/cmip6-en/the-ssp-scenarios)). The Annual Maximum Near-Surface Air Temperature serves as an essential metric for assessing and understanding future climate for the different emissions scenario, and is a valuable tool for climate scientists, researchers, and policymakers. Monitoring and projecting near surface air temperature over time allows for the identification of trends and variations in temperature extremes, which is crucial for understanding the impact of climate change on local and global climates.
```
Comment (j08lue) [inline datasets/cmip6-tasmax-monthly-annual-max.data.mdx:89]:
```suggestion
    This dataset provides the Annual Maximum Near-Surface Air Temperature from 1950 - 2100. The projections are provided for the four different SSP standard scenarios ([SSP126, SSP245, SSP370, SSP585](https://www.dkrz.de/en/communication/climate-simulations/cmip6-en/the-ssp-scenarios)). The Annual Maximum Near-Surface Air Temperature serves as an essential metric for assessing and understanding future climate for the different emissions scenarios, and is a valuable tool for climate scientists, researchers, and policymakers. Monitoring and projecting near surface air temperature over time allows for the identification of trends and variations in temperature extremes, which is crucial for understanding the impact of climate change on local and global climates.
```

File: datasets/cmip6-tasmax-monthly-annual-max.data.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  - name: Product Type
+    values:
+      - Model Output
+infoDescription: |
+  ::markdown
+    - **Temporal Extent:** 1950 - 2100
+    - **Temporal Resolution:** Annual
+    - **Spatial Extent:** Global 
+    - **Spatial Resolution:** 0.25 degrees x 0.25 degrees
+    - **Data Units:** F
```
Comment (j08lue) [inline datasets/cmip6-tasmax-monthly-annual-max.data.mdx:28]:
```suggestion
    - **Data Units:** °F
```

File: datasets/cmip6-tasmax-monthly-annual-max.data.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      - "#FAED2D"
+      - "#FFCE0A"
+      - "#FFB100"
+      - "#FE9900"
+      - "#FC7F00"
+    info:
+      source: NASA
+      spatialExtent: Global
+      temporalResolution: Annual
+      unit: Days
```
Comment (j08lue) [inline datasets/cmip6-tasmax-monthly-annual-max.data.mdx:74]:
```suggestion
      unit: °F
```

**Review by j08lue — CHANGES_REQUESTED** [review]
Reviewed the new dataset:

https://deploy-preview-131--earth-information-center.netlify.app/data-catalog/cmip6-climdex-tasmax-yearly-median

Made a few copy edits. 

Looks good in terms of functionality.

There are a few unrelated changes in this PR. Perhaps this needs to be rebased to a more recent `develop` or so?

**Review by agurvich — APPROVED** [review]

**Pull request merged**